billion-scale-image-search/src/main/application/schemas/image.sd

```diff
@@ -0,0 +1,107 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+schema image {
+
+  document image {
+
+    # Don't need to guess language
+    field language type string {
+      indexing: "en" | set_language
+    }
```

Comment:
https://docs.vespa.ai/en/linguistics.html

I think you can set this outside the document clause, but not sure if it makes a difference